basis/windows/ole32/ole32-tests.factor

```diff
@@ -29,3 +30,7 @@ IN: windows.ole32.tests
 [ "{01234567-89ab-cdef-0123-456789abcdef}" ]
 [ "{01234567-89ab-cdef-0123-456789abcdef}" string>guid guid>string ]
 unit-test
+
+{ f } [
+    create-guid 10 iota [ drop create-guid guid= ] with any?
```

Comment:
For what it's worth, another way to do this is make 10 of them and check there are no duplicates:

```factor
{ 0 } [ 10 [ create-guid ] replicate duplicates length ] unit-test
```